DuplicateSymbols.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace LineAnalysis
+{
+    /// <summary>
+    ///  This class is about search dublicate symbols in a string.
+    /// </summary>
+    class DuplicateSymbols
+    {
+        private string userString;
+
+        public DuplicateSymbols(string args)
+        {
+            userString = args;
+        }
+
+        /// <summary>
+        /// This method researches users string to found maximal series of duplicate symbols.

```

Comment:
"This method researches..." bad comment, method do not research, it find or calculate